src/Database/ODBC/Internal.hs

```diff
@@ -605,6 +607,40 @@ getData dbc stmt i col =
                    (fmap fromIntegral (odbc_TIME_STRUCT_hour datePtr)) <*>
                    (fmap fromIntegral (odbc_TIME_STRUCT_minute datePtr)) <*>
                    (fmap fromIntegral (odbc_TIME_STRUCT_second datePtr))))
+     | colType == sql_ss_timestampoffset ->
+       withCallocBytes
+         20
+         (\datePtr -> do
+            mlen <-
+              getTypedData
+                dbc
+                stmt
+                sql_c_binary
+                i
+                (coerce datePtr)
+                (SQLLEN 20)
```

Comment:
Also here, shouldn't it be 10 bytes?

Reply:
I used 20 bytes here for the same [reason as above](#discussion_r283809972).

I assumed that the two sizes had to be the same, but I admit that this is where I was guessing.